build.gradle

```diff
@@ -53,8 +51,6 @@ dependencies {
     implementation('org.springdoc:springdoc-openapi-ui:1.6.2')
 
     implementation('com.github.ben-manes.caffeine:caffeine:3.0.5')
-    implementation("org.apache.camel:camel-spring-boot-starter:${camelVersion}")
-    implementation("org.apache.camel:camel-aws:${camelVersion}")
     implementation('com.amazonaws:aws-java-sdk:1.12.131')
```

Comment:
Can this go? its hundreds of Mb
See https://github.com/UKHomeOffice/hocs-casework/pull/540
<img width="589" alt="Screenshot 2021-12-22 at 22 31 38" src="https://user-images.githubusercontent.com/24651657/147162481-85d88f4c-c0bc-4027-a03f-5afa71e37dcd.png">

Reply:
Good catch, yes will sort tomorrow...